### ISSUE TITLE
IOS - Territory filter is required on all requests in a future release.

### ISSUE THREAD

**Issue by egem** [j3k0/cordova-plugin-purchase#1608]
I have got just a question instead of the bug report. I am not sure whether I can ask question here, so if I shouldn't please let me know.

Apple sent a mail that

We’re reaching out to let you know that the subscriptionPricePoints endpoint now requires a territory relationship filter. If you use the App Store Connect API to configure your subscriptions, please note that you'll be required to define this filter in an upcoming release.

I wonder whether it was already implemented and already have been using territory filter or if it is not implemented, have you got any plan to implement? 

Thanks in advance for your time.

**Comment by jaroshevskii** [comment]
The AppStore API has been updated, now when requesting, you must always specify a third-party filter. This endpoint is not used anywhere in the clean surf code.

**Everyone received this message**, so don't worry if you don't use the AppStore API.

**Comment by egem** [comment]
Thank you for the information.  I am closing the issue.

**Comment by nathaniel-eguna** [comment]
I also received this email earlier. The email address is developer@insideapple.apple.com, Is this address legit or just spam?

**Comment by Trulsmatias** [comment]
@jaroshevskii just to clarify: is this already ok and not something we need to think about if we only are using this plugin for purchases and not the App Store API directly?

**Comment by jaroshevskii** [comment]
@Trulsmatias I don't use this plugin so I can't say anything about it)

**Comment by Trulsmatias** [comment]
Thank you for you answer @jaroshevskii 

Doesn't this mean this issue should be reopened @egem? I don't see why this is resolved anymore.

**Comment by egem** [comment]
Yes, you are right @Trulsmatias. I am openning this issue again.
